From 4aa2cdc5a29ce2587180c464804b1362b8404e2f Mon Sep 17 00:00:00 2001
From: container-user <container-user@example.com>
Date: Thu, 21 Mar 2024 11:56:11 +0100
Subject: [PATCH] Add gitignore

---
 .gitignore | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..34f895a
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,10 @@
+# Prerequisites
+*.d
+
+# Object files
+*.o
+*.elf
+
+# Copied source files
+zynqmp_pmufw_bsp/psu_pmu_0/include/
+zynqmp_pmufw_bsp/psu_pmu_0/lib/
-- 
2.25.1

