From 1789e17f6b17b109e7bd19644ecd2d9d217b8e4c Mon Sep 17 00:00:00 2001
From: container-user <container-user@example.com>
Date: Thu, 21 Mar 2024 11:49:49 +0100
Subject: [PATCH] Add gitignore

---
 .gitignore | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..fef82f6
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,10 @@
+# Prerequisites
+*.d
+
+# Object files
+*.o
+*.elf
+
+# Copied source files
+zynqmp_fsbl_bsp/psu_cortexa53_0/include/
+zynqmp_fsbl_bsp/psu_cortexa53_0/lib/
-- 
2.25.1

