From 8139ae65fca893b68c5270e3a6209e75587472da Mon Sep 17 00:00:00 2001
From: "marvin.fuchs" <marvin.fuchs@kit.edu>
Date: Wed, 12 Jun 2024 18:56:15 +0200
Subject: [PATCH] Add directory with generated files to gitignore

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index fd44aa1770..be9b813fd8 100644
--- a/.gitignore
+++ b/.gitignore
@@ -57,6 +57,7 @@ fit-dtb.blob*
 /spl/
 /tpl/
 /defconfig
+/tools/boot
 
 #
 # Generated include files
-- 
2.25.1

